### PR TITLE
Items additions/ removal of nonexistent ones

### PR DESCRIPTION
**Pull request by IKnewOne**
Changes quest coins name in browser+ adds wooden plank
To be updated

### CODE REVIEW

**Comment by NetroScript** [comment]
You don't have to revert that change, like I said it would be best removing all items where another instance with ":1-3" exists. I also want to add that Fake Coins and Coins are 2 different items. If you add both quest_fake_coins and coins to you inventory, you will see those are 2 different items, the first is the quest item, the second is no idea

**Comment by NetroScript** [comment]
If you want I can do the part with items with qualities and those with none automated (removing the base item and adding all possible qualities)

**Comment by IKnewOne** [comment]
Well, it's up for you to decide. I'm just trying to help

File: data/html/items.json [hunk 1]
```diff
@@ -516,6 +516,7 @@
 		"rightful_citizen_paper",
 		"restoration_instrument",
 		"wooden_plank"
```
Comment (NetroScript) [inline data/html/items.json]: Comma missing

**Comment by IKnewOne** [comment]
Tho i don't know how to add warning
And where, if adding is a matter of css/j

**Comment by NetroScript** [comment]
No problem, I will add the warning myself - do you want to change anything else or did you do the changes you noticed about items? If so I will merge it

**Comment by IKnewOne** [comment]
I've done all i noticed for now. If anything- i can just add another pull request

**Pull request merged**